tests/rule_based_profiler/test_profiler.py

```diff
@@ -258,3 +259,57 @@ def test_batches_are_accessible(
             )
         )
         assert metric_value_set == {"category0", "category1", "category2"}
+
+
+def test_profile_includes_citations(
```

Comment:
I used the `alice` fixtures here in order to create a `DataContext`, instantiate a `Profiler`, and run `profile()`. Did we want to use an entirely different set of fixtures here and reserve the named ones for `test_profiler_user_workflows.py`?

Reply:
@cdkini I think that `alice` is fine; the only thing is that we will likely refactor and clean up the way the tests and the fixtures are set up and stored at some point, and hence will potentially have to rearrange how these are utilized, too.